packages/orchestration/src/exos/portfolio-holder-kit.js

```diff
@@ -43,7 +43,7 @@ const preparePortfolioHolderKit = (zone, { asVow, when }) => {
     'PortfolioHolderKit',
     {
       invitationMakers: M.interface('InvitationMakers', {
-        MakeInvitation: M.call(
+        Proxying: M.call(
```

Comment:
Wb `Proxy`, verb, instead of a present participle?

Reply:
That was my first thought too but since [Proxy is a global object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) I thought it would be confusing.